tensorflow/go/attrs.go

```diff
@@ -170,8 +173,14 @@ func listAttribute(op *Operation, cname *C.char, meta C.TF_AttrMetadata) (interf
 			}
 			// A []C.int64_t slice backed by C memory.
 			// See: https://github.com/golang/go/wiki/cgo#turning-c-arrays-into-go-slices
-			slice := (*[1 << 30]C.int64_t)(unsafe.Pointer(dim))[:numDim:numDim]
-			list[i] = makeCShape(slice)
+			if is64Bit {
```

Comment:
Do we need this `is64Bit` check at all?
Given than `numDim` is unlikely to be anywhere close to 1 << 30, irrespective of architecture, I wonder if we can always use `[1 << 27]` (with may a comment saying "using 27 instead of 30 so that this works on 32-bit architectures as well. Should be fine since the dimensions array will not be anywhere close to 2^30 elements long)?